lib/dune

```diff
@@ -0,0 +1,10 @@
+(library
+  (name core_lwt)
+  (public_name core_lwt)
```

Comment:
please, update `public_name` here: there should be `core-lwt`. `name` stanza is correct.

Reply:
No, it should be `core_lwt`! Read the comment above.